content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Manual Rotation of CA certificates
+content_template: templates/task
+reviewers:
+- 
+---
+
+{{% capture overview %}}
+
+This page shows how to rotate CA certificates manually.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Read about the authentication in kubernetes [here](/docs/reference/access-authn-authz/authentication)
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+There are two possible ways of rotating or replacing the CA certificates.
+
+## Replace The Cluster (Recommended)
+
+This is by far the safest approach to rotate or replace the CA certificates.
+
+1. Create a new cluster with new CA certificates.
+
+2. Migrate the workloads to the new cluster.
+
+3. Remove or destroy the old cluster.
+
+## Rotate the CA certificates Manually
+
+{{< caution >}}
+The rotation or replacement of CA Certificates is not supported directly by the ```kubeadm``` at this time and hence this documentation has to be followed with caution. Please make sure to take necessary backups of your certificate directory along with confs and any other necessary files.
```

Comment:
@abhiTamrakar 

CA rotation is arguably out of scope for core-kubeadm.
maybe we can add it as part of the "kubeadm operator". a search query in k/kubeadm will let you find the tracking issue for that.

i'd prefer if we remove the kubeadm mention here. maybe link from the kubeadm docs to this page if it's approved.

Reply:
@neolit123 I am partly agreed here. I was thinking about people like me who struggled to find CA rotation was not supported by kubeadm or any kubernetes sub-component. We need some place where we mention what we do not support at present and what all is in our pipeline.

I would request to have it there for time being and once kubeadm operator #1698 (I guess) becomes ready, this documentation would be updated.

Reply:
you can add a new section here called "CA rotation":
https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/

that explains with once sentence that kubeadm does not support CA rotation out of the box and link to this new page.

once the "operator" supports it, we can add this detail as well in the kubeadm certs management page.

Reply:
done, moved the line and added to the page you have mentioned.

Reply:
@neolit123 should we include `kubeadm init phase` commands as an alternative in the document. Using kubeadm, if we upload certs or re-create `kubeadm-certs` secret then `kubeadm init phase kubeconfig` and `kubeadm init phase kubelet-start` would actually help by passing many manual patches in conf files and kubelet certs along with human errors.

Reply:
for the new page, it should remain kubeadm agnostic without any kubeadm commands.
`kubeadm-certs` is a kubeadm implementation detail, so possibly you can mention kubeadm specifics in the "CA rotation" section of:
https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/

but also i would prefer if we keep the kubeadm documentation with minimal examples and not expand too much in the lines of a exact step-by-step tutorial, because kubeadm HA is still Beta and has moving parts.

Reply:
okay.